src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -189,8 +189,10 @@ export class SchoolSelectField extends React.Component {
                     <label
                       id={`institution-${index}-label`}
                       htmlFor={`page-${currentPageNumber}-${index}`}>
-                      <span className="institution-name">{name}</span>
-                      <span className="institution-city-state">{`${city}, ${state}`}</span>
+                      {name && <span className="institution-name">{name}</span>}
+                      {street && <span className="institution-street">{street}</span>}
+                      {(city || state) && <span className="institution-city-state">{`${city && city}${city && state && ', '}${state && state}`}</span>}
```

Comment:
Will do: 
- Salt Lake City, UT
- Salt Lake City
- UT
- USA

but will never do: 
- Salt Lake City,
- , UT
- Salt Lake City, UT USA

Reply:
Should we consider moving this logic into a helper function with its own unit tests, it seems like a very helpful utility to include in `platform` eventually.

Reply:
That's probably a good idea. I'll make a ticket and let's discuss how to make it discoverable.